app/admin/store_products.rb

```diff
@@ -49,7 +49,8 @@ def update_image
       row :clicks
       row :link
       row :image do |product|
-        image_tag(url_for(product.image)) if product.image.attached?
+        image_tag(url_for(product.image),
+                  size: 250, class: "aa-product__image") if product.image.attached?
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.